.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+This PR closes #{issue_number}.
+
+<!-- 
+    Write a short description to explain changes that are not mentioned in the initial issue.
```

Comment:
Do we have such cases? Usually, we create separate PRs.

Reply:
Yep, we have such things all the time. Mostly, because issues are not well-described :)